src/form/death/index.ts

```diff
@@ -0,0 +1,266 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * OpenCRVS is also distributed under the terms of the Civil Registration
+ * & Healthcare Disclaimer located at http://opencrvs.org/license.
+ *
+ * Copyright (C) The OpenCRVS Authors. OpenCRVS and the OpenCRVS
+ * graphic logo are (registered/a) trademark(s) of Plan International.
+ */
+
+import {
+  exactDateOfBirthUnknown,
+  getAgeOfIndividualInYears,
+  getMaritalStatus,
+  registrationEmail,
+  registrationPhone,
+  seperatorDivider,
+  getNationalID
+} from '../common/common-optional-fields'
+import {
+  getGender,
+  getBirthDate,
+  getFamilyNameField,
+  getFirstNameField,
+  getNationality,
+  otherInformantType
+} from '../common/common-required-fields'
+import {
+  deathInformantType,
+  getCauseOfDeath,
+  getCauseOfDeathMethod,
+  getDeathDate,
+  getDeathDescription,
+  getMannerOfDeath,
+  getPlaceOfDeathFields
+} from './required-fields'
+import { formMessageDescriptors } from '../common/messages'
+import { Event, ISerializedForm } from '../types/types'
+import {
+  getNationalIDValidators,
+  hideIfNidIntegrationEnabled,
+  informantBirthDateConditionals,
+  informantFamilyNameConditionals,
+  informantFirstNameConditionals,
+  exactDateOfBirthUnknownConditional,
+  isValidBirthDate
+} from '../common/default-validation-conditionals'
+import { documentsSection, registrationSection } from './required-sections'
+import {
+  deceasedNameInEnglish,
+  informantNameInEnglish
+} from '../common/preview-groups'
+import { certificateHandlebars } from './certficate-handlebars'
+
+// ======================= FORM CONFIGURATION =======================
+
+// A REGISTRATION FORM IS MADE UP OF PAGES OR "SECTIONS"
+
+// A "SECTION" CAN BE SPLIT OVER MULTIPLE PAGES USING "GROUPS" ALTHOUGH THIS MAY BE DEPRECATED
+
+export const deathForm = {
+  sections: [
+    registrationSection, // REQUIRED HIDDEN SECTION CONTAINING IDENTIFIERS
+    {
+      id: 'information',
+      viewType: 'form',
+      name: {
+        defaultMessage: 'Information',
+        description: 'Form section name for Information',
+        id: 'form.section.information.name'
+      },
+      groups: [
+        {
+          id: 'information-group',
+          title: {
+            defaultMessage:
+              'Introduce the death registration process to the informant',
+            description: 'Event information title for the birth',
+            id: 'register.eventInfo.death.title'
+          },
+          conditionals: [
+            {
+              action: 'hide',
+              expression: 'window.config.HIDE_DEATH_EVENT_REGISTER_INFORMATION'
```

Comment:
@euanmillar Do we even this configuration option because we can just not add the section entirely?